### PR TITLE
[3.1.0] Update customizing dev portal docs

### LINKED ISSUE #1487: [APIM-3.1.0] Updates to the UI customization pages
**Description:**
The following page [1] which contains details about styling the Logo and the Header of the Developer Portal needs to mention in which file the custom configurations should be applied. The customizations should be applied in the following file. 

`<API-M_HOME>/repository/deployment/server/jaggeryapps/devportal/site/public/theme/defaultTheme.js`

In the following document [2], it's mentioned that the changes should be done in the above file. But it easily misleads someone following the document to update the following file instead. 

`<API-M_HOME>/repository/deployment/server/jaggeryapps/devportal/source/src/defaultTheme.js`

It would be better to add a note mentioning the above file can be used as a reference and the actual customization should be applied in the defaultTheme.js file in the theme directory. 

[1]. https://apim.docs.wso2.com/en/latest/develop/customizations/customizing-the-developer-portal/styling-the-logo-and-header/

[2]. https://apim.docs.wso2.com/en/latest/develop/customizations/customizing-the-developer-portal/overriding-developer-portal-theme/

**Affected Product Version:**: APIM-3.1.0

### PR DESCRIPTION
**Pull request by Jayanie**
## Purpose
 
- This PR fixes #1487

### CODE REVIEW

**Review by pubudu538 — APPROVED** [review]

File: en/docs/develop/customizations/customizing-the-developer-portal/styling-the-logo-and-header.md [hunk 1]
```diff
@@ -1,9 +1,16 @@
...
 
 The header section can be customized to match different design needs. Following is the default look and the configuration.
+The default header of the API Manager Developer Portal is shown below.
 
  ![changing the logo and header]({{base_path}}/assets/img/learn/changing-the-logo-and-header1.png) 
 
+!!! note
+    -   You can find the default theme in the following location
+    `<APIM_HOME>/repository/deployment/server/jaggeryapps/devportal/source/src/defaultTheme.js`
+    -   Using the above file as reference you can override the parameters defined in the above file by altering the `defaultTheme.js` file in following location
```
Comment (Sarangan0219) [inline en/docs/develop/customizations/customizing-the-developer-portal/styling-the-logo-and-header.md:11]: Shall we rephrase this as,
 
**Using the above file as a reference, you can override the parameters defined in the above file, by altering the `defaultTheme.js` file in the following location.**

Comment (Jayanie) [reply en/docs/develop/customizations/customizing-the-developer-portal/styling-the-logo-and-header.md:11]: Fixed with 10584cd96c089244ce8914132e6b047ff59d5453

Comment (Jayanie) [reply en/docs/develop/customizations/customizing-the-developer-portal/styling-the-logo-and-header.md:11]: This is fixed in the latest
